templates/accounts/edit.html

```diff
@@ -66,8 +66,9 @@ <h3>Delete account</h3>
 
 <script>
   $(function() {
-    unsecureImageCheck($('#id_profile-about'));
-    unsecureImageCheck($('#id_profile-signature'));
+      {% for field in profile_form.get_img_fields %}
```

Comment:
I guess this should be renamed to `profile_form.get_img_check_fields` now?

Reply:
Updated, forgot to change name before